### DISCUSSION TITLE
Load times

### DISCUSSION THREAD

**Discussion by patriksvensson** [JanDeDobbeleer/oh-my-posh#3048] [Troubleshoot]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

I was talking with a friend over a beer and he's a big fan of oh-my-posh, but he experiences a lot of lag between prompts on a certain computer. What information can he provide to make a good bug report around this?

I will update this issue once I know what is expected.

### Theme

All themes.

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
No log output at this time.
```

cc: @monophonik

**Comment by JanDeDobbeleer** [comment]
@patriksvensson couple of things to validate:
- make sure to be on the latest version and use official installation means only due to getting a signed executable 
- if it's slow, likely Windows (Defender/antivirus) is still blocking the execution 
- if an exclusion also doesn't help, in the path where it's slow, provide the output of `oh-my-posh debug`

If your friend is still on the PowerShell module, make sure to [migrate](https://ohmyposh.dev/docs/migrating) first.
